native/packages/react-native-bpk-component-animate-height/src/BpkAnimateHeight.js

```diff
@@ -34,92 +33,115 @@ const {
   animationDurationSm,
 } = tokens;
 
+const collapsedHeight = 0.01;
+
 class BpkAnimateHeight extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
-      expanded: null,
-      expandedHeight: null,
-      collapsedHeight: 0,
-      height: new Animated.Value(1),
-      heightSet: false,
+      expanded: this.props.expanded,
+      height: new Animated.Value(0.01),
```

Comment:
Can we make this use the constant defined above, seeing as the value is the same?